test/parser.js

```diff
@@ -0,0 +1,146 @@
+
```

Comment:
Let's use https://github.com/standard/standard for the app for consistent styling and readability.